custom_components/bosch_alarm/alarm_control_panel.py

```diff
@@ -38,18 +37,19 @@
 })
 
 class AreaAlarmControlPanel(AlarmControlPanelEntity):
-    def __init__(self, panel, arming_code, area_id, area, unique_id):
-        self._panel = panel
+    def __init__(self, connection, arming_code, area_id, area, unique_id):
```

Comment:
There's a few places that still use `data` as the variable name, and elsewhere we use `connection`, but `connection.panel` reads kind strange. How about `panel_conn` or `panel_cnt` or something like that?